server/java/src/test/java/com/godaddy/asherah/grpc/AppEncryptionImplTest.java

```diff
@@ -0,0 +1,210 @@
+package com.godaddy.asherah.grpc;
```

Comment:
I'm seeing a lot of linter errors with these test files, mainly around star imports and indentation. If tests are meant to be exempt we should probably update `checkstyle.xml` to suppress these explicitly.